.github/workflows/repo-health-job.yml

```diff
@@ -0,0 +1,103 @@
+      ONLY_CHECK_THIS_REPOSITORY:
+        description: "If you only want to run repo health on one repository, set this to org/name of said repository."
+        type: string
+        default: ""
+      REPORT_DATE:
+        description: "The date for which repo health data is required.(format: YYYY-MM-DD)"
+        type: string
+        default: ""
+      REPO_HEALTH_GOOGLE_CREDS_FILE:
+        description: "Link to the repo health credentials file"
+        type: string
+        default: ""
+      REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL:
+        description: "URL for the repo health spreadsheet to populate"
+        type: string
+        default: ""
+      REPO_HEALTH_REPOS_WORKSHEET_ID:
+        description: "Repo health repositories sheet ID"
+        type: number
+        default: 0
+
+jobs:
+  repo_health_check:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: ["3.8"]
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: setup python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      
+      - name: Install pip-tools
+        run: python3 -m pip install pip-tools
+      
+      - name: Clone edx-repo-health repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/edx-repo-health
+          path: edx-repo-health
+          ref: ${{ inputs.EDX_REPO_HEALTH_BRANCH }}
+      
+      - name: Clone testeng-ci repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/testeng-ci
+          path: testeng-ci
+          ref: "master"
+      
+      - name: Clone repo-tools repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/repo-tools
+          path: repo_tools
+          ref: master
+      
+      - name: Clone repo-health-data repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/repo-health-data
+          path: repo-health-data
+          ref: master
+          token: ${{ secrets.REPO_HEALTH_BOT_TOKEN}}
+
+      - name: Run repo health org script
+        env:
+          ORG_NAMES: ${{ inputs.ORG_NAMES }}
+          # REPORT_DATE: ${{ inputs.REPORT_DATE }}
+          GITHUB_TOKEN: ${{ secrets.REPO_HEALTH_BOT_TOKEN }}
+          GITHUB_USER_EMAIL: ${{ secrets.REPO_HEALTH_BOT_EMAIL }}
+          EDX_REPO_HEALTH_BRANCH: ${{ inputs.EDX_REPO_HEALTH_BRANCH }}
+          # ONLY_CHECK_THIS_REPOSITORY: ${{ inputs.ONLY_CHECK_THIS_REPOSITORY }}
+          REPO_HEALTH_GOOGLE_CREDS_FILE: ${{ inputs.REPO_HEALTH_GOOGLE_CREDS_FILE }}
+          # REPO_HEALTH_REPOS_WORKSHEET_ID: ${{ inputs.REPO_HEALTH_REPOS_WORKSHEET_ID }}
+          REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL: ${{ inputs.REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL }}
+        run: |
+          curl -o $PWD/repo-health-script.sh https://raw.githubusercontent.com/UsamaSadiq/openedx-github/update-repo-health-job/scripts/repo-health-script.sh
```

Comment:
We've checked out this repo, so this line can run the local file, I think?

Reply:
When the reusable/shared workflow was triggered from a workflow under different organisation, it only copied the workflow file and couldn't access the script file directly. 
We had two options of either going with creating a composite workflow or downloading the script to be used so we went with the later approach since it was simple and easier to understand.

Reply:
If we never call that script directly from this repo, would it make sense to move it to `edx-repo-health` which is checked out into a local directory earlier in the workflow?